core/arch/arm/kernel/proc_a32.S

```diff
@@ -89,6 +89,15 @@ UNWIND(	.fnstart)
 UNWIND(	.fnend)
 END_FUNC cpu_spin_unlock
 
+/* int cpu_mmu_enabled(void) - check MMU enabled or not */
+FUNC cpu_mmu_enabled , :
```

Comment:
@jenswi-linaro 
Which file do you think I should put cpu_mmu_enabled in? Or I drop this and directly use read_sctlr in my driver?